curriculumBuilder/settings.py

```diff
@@ -14,7 +14,8 @@
 from urlparse import urlparse
 
 BASE_DIR = os.path.dirname(os.path.dirname(__file__))
-I18N_STATIC_DIR = os.path.join(BASE_DIR, "i18n", "static")
+I18N_DIR = os.path.join(BASE_DIR, "i18n")
+I18N_STATIC_DIR = os.path.join(I18N_DIR, "static")
```

Comment:
I don't think this is what we want; I'd expect this to end up being a value like `/path/to/the/project/curriculumBuilder/i18n/static`

Reply:
Do we want this to be a relative path?

Reply:
I believe that the `i18n_dir` and `static_dir` helper methods are used by the I18nFileWrapper; since they're now referencing these settings, the settings need to be compatible with non-local-filesystem storage solutions